docker/Makefile

```diff
@@ -4,17 +4,21 @@ version=$(shell git describe --tags --abbrev=0)
 tag=latest
 cmd=
 user=directus
-registry=docker.io
-repository=directus/directus
+registry=gcr.io
+repository=uncoil-io/directus
+tmp_package_json=$(mktemp)
 
 .PHONY: build-images
 
 build-images:
-	npm run build
-	npm run pack
-	docker build \
+	pnpm install
```

Comment:
Modified the build commands according to https://github.com/directus/directus/discussions/15199#discussioncomment-3443630

Reply:
Ah nice, that's a good idea